shared/src/persistence/dynamo/helpers/isCurrentColorActive.js

```diff
@@ -0,0 +1,20 @@
+const client = require('../../dynamodbClientService');
+
+/**
+ * Checks to see if the environment currentColor matches that of the deploy table
+ *
+ * @param {object} applicationContext The applicationContext
+ * @returns {Promise} which resolves to whether or not the current color matches what's in the deploy table
+ */
+exports.isCurrentColorActive = async applicationContext => {
+  const { current: currentColor } = await client.get({
```

Comment:
Nit: dynamoDbClientService has a `getFromDeployTable` function, perhaps that could be used here?